bfasst/flows/flow_descriptions.yaml

```diff
@@ -81,6 +81,12 @@ flows:
   - fasm2bels
   - rapidwright
 
+- name: VivadoReimplCompare
```

Comment:
Maybe let's call this flow something else?  We should come up with a cool name for the project.